src/EnergyPlus/SetPointManager.hh

```diff
@@ -121,37 +121,6 @@ namespace SetPointManager {
         Num
     };
 
-    int constexpr NumValidCtrlTypes = 9;
-
-    inline const char *controlTypeName(CtrlVarType cvt)
-    {
-        switch (cvt) {
-        case CtrlVarType::Temp:
-            return "Temperature";
-        case CtrlVarType::MaxTemp:
-            return "MaximumTemperature";
-        case CtrlVarType::MinTemp:
-            return "MinimumTemperature";
-        case CtrlVarType::HumRat:
-            return "HumidityRatio";
-        case CtrlVarType::MaxHumRat:
-            return "MaximumHumidityRatio";
-        case CtrlVarType::MinHumRat:
-            return "MinimumHumidityRatio";
-        case CtrlVarType::MassFlow:
-            return "MassFlowRate";
-        case CtrlVarType::MaxMassFlow:
-            return "MaximumMassFlowRate";
-        case CtrlVarType::MinMassFlow:
-            return "MinimumMassFlowRate";
-        case CtrlVarType::Invalid:
-            return "*UNKNOWN*";
-        default:
-            assert(false);
-        }
-        return "*UNKNOWN*"; // not sure how we would get here, the switch block cases are exhaustive
-    }
-
```

Comment:
we replaced this with `constexpr std::array<std::string_view, static_cast<int>(CtrlVarType::Num)> controlTypeName` as requested.